src/test/resources/application.yml

```diff
@@ -3,5 +3,8 @@ server:
 endpointUrl:
   retrieve: http://localhost:${server.port}/mis/product/
   index: http://search:8080/index/
+endpoints:
+  index:
+    version: testIndexApiVersion
```

Comment:
❓ Is this supposed to be a placeholder `${testIndexApiVersion}` or an actual value for the version?

Reply:
This is the actual value.